internal/cmd/plugin/command_install.go

```diff
@@ -184,25 +185,22 @@ func installPlugin(manifest *Manifest, repositoryDir, installDir string) error {
 }
 
 func getLanguage(manifest *Manifest) (string, *version.Version) {
-	if manifest == nil {
+	if manifest == nil || len(manifest.Dependencies) == 0 {
 		return "", nil
 	}
 
-	if manifest.Dependencies == "" {
-		return "", nil
-	}
-
-	dependencySlice := strings.Split(strings.ToLower(manifest.Dependencies), " ")
-	if len(dependencySlice) == 1 {
-		return dependencySlice[0], nil
+	primaryDependency := manifest.Dependencies[0]
```

Comment:
Do we plan to support multiple dependencies in the future?

Reply:
I was thinking that we basically already can, because someone could write a plugin that calls other programs (e.g. a plugin in Go that uses `exec`).

Maybe we should enforce instead that there's only one dependency?